samples/ts/21.react-media-template/src/components/LiveNotifications.tsx

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { useEffect, useState, useRef, FC } from "react";
+import { getFlexColumnStyles } from "../styles/layouts";
+import { mergeClasses } from "@fluentui/react-components";
+import { getLiveNotificationStyles, getPillStyles } from "../styles/styles";
```

Comment:
add an index.ts exporting layouts and styles, then the following can be done:
```suggestion
import { getLiveNotificationStyles, getPillStyles } from "../styles";
```

Reply:
Probably I need to do this for the other samples - I'll handle that.